python/ray/data/_internal/execution/backpressure_policy/concurrency_cap_backpressure_policy.py

```diff
@@ -17,66 +19,28 @@ class ConcurrencyCapBackpressurePolicy(BackpressurePolicy):
     """A backpressure policy that caps the concurrency of each operator.
 
     The concurrency cap limits the number of concurrently running tasks.
-    It will be set to an intial value, and will ramp up exponentially.
 
     The concrete stategy is as follows:
-    - Each PhysicalOperator is assigned an initial concurrency cap.
+    - Each PhysicalOperator is assigned a concurrency cap.
     - An PhysicalOperator can run new tasks if the number of running tasks is less
       than the cap.
-    - When the number of finished tasks reaches a threshold, the concurrency cap will
-      increase.
-    """
-
-    # Following are the default values followed by the config keys of the
-    # available configs.
-    # Use `DataContext.set_config` to config them.
 
-    # The intial concurrency cap for each operator.
-    INIT_CAP = 4
-    INIT_CAP_CONFIG_KEY = "backpressure_policies.concurrency_cap.init_cap"
-    # When the number of finished tasks reaches this threshold, the concurrency cap
-    # will be multiplied by the multiplier.
-    CAP_MULTIPLY_THRESHOLD = 0.5
-    CAP_MULTIPLY_THRESHOLD_CONFIG_KEY = (
-        "backpressure_policies.concurrency_cap.cap_multiply_threshold"
-    )
-    # The multiplier to multiply the concurrency cap by.
-    CAP_MULTIPLIER = 2.0
-    CAP_MULTIPLIER_CONFIG_KEY = "backpressure_policies.concurrency_cap.cap_multiplier"
+    NOTE: Only support setting concurrency cap for `TaskPoolMapOperator` for now.
```

Comment:
What's our motivation for not adding it to `ActorPoolMapOperator`? Is it because there's a separate code path for controlling actor concurrency?

Reply:
> Is it because there's a separate code path for controlling actor concurrency?

Yes, the concurrency cap back pressure policy here is not useful given we already have internal scaling up and down in `ActorPoolMapOperator`.

Reply:
also add this to the comment? and maybe also leave a TODO to consolidate this policy with the actor pool concurrency.